tests/unit/test-util.js

```diff
@@ -86,15 +97,23 @@ const testSingleResource = (serializedResource, resourceType, resourceId, nested
  * @param {object} rawResources Raw resources to be used in test.
  * @param {*} resourceType The type of the resource.
  * @param {*} resourceKey The id of the resource.
+ * @param {object} relationships The relationships of the resource.
  */
-const testMultipleResources = (serializedResources, rawResources, resourceType, resourceKey) => {
+const testMultipleResources = (serializedResources,
+  rawResources, resourceType, resourceKey, relationships) => {
   expect(serializedResources).to.have.all.keys('data', 'links');
   expect(serializedResources.data).to.be.an('array');
-  _.zipWith(serializedResources.data, rawResources, (a, b) => {
-    expect(a).to.deep.equal(resourceSchema(resourceType,
-      b[resourceKey],
-      _.omit(b, resourceKey)).data);
-  });
+  for (let i = 0; i < serializedResources.data.length; i += 1) {
```

Comment:
You should be able to use [`_.zipWith`](https://lodash.com/docs/4.17.15#zipWith) here